packages/framework/pr_tools/PullRequestLinuxDriver.sh

```diff
@@ -170,7 +170,7 @@ sig_merge_old=$(get_md5sum ${REPO_ROOT:?}/packages/framework/pr_tools/PullReques
 if [[ ${on_kokkos_develop} == "1" ]]; then
     message_std "PRDriver> --kokkos-develop is set - setting kokkos and kokkos-kernels packages to current develop and pointing at them"
     "${SCRIPTPATH}"/SetKokkosDevelop.sh
-    extra_configure_args="\"-DKokkos_SOURCE_DIR_OVERRIDE:string=kokkos;-DKokkosKernels_SOURCE_DIR_OVERRIDE:string=kokkos-kernels\"${extra_configure_args:+;${extra_configure_args}}"
+    extra_configure_args="-DKokkos_SOURCE_DIR_OVERRIDE:string=kokkos;-DKokkosKernels_SOURCE_DIR_OVERRIDE:string=kokkos-kernels${extra_configure_args:+;${extra_configure_args}}"
```

Comment:
```suggestion
    extra_configure_args="-DKokkos_SOURCE_DIR_OVERRIDE:string=kokkos;-DKokkosKernels_SOURCE_DIR_OVERRIDE:string=kokkos-kernels;${extra_configure_args:+;${extra_configure_args}}"
```

I'm probably misinterpreting how the string is filled in after "kokkos-kernels", but wanted to check if a semi-colon needed before the ${extra...} arg?

Reply:
It is, but it's injected within the shell variable expansion (look a little bit further on the line)